cmd/flags.go

```diff
@@ -187,18 +187,27 @@ func (f *capabilitiesFlag) Type() string {
 }
 
 func (f *capabilitiesFlag) String() string {
-	return f.path
+	return f.pathOrVersion
 }
 
 func (f *capabilitiesFlag) Set(s string) error {
-	f.path = s
-	fd, err := os.Open(s)
-	if err != nil {
+	f.pathOrVersion = s
+	fd, errPath := os.Open(s)
+	if errPath == nil {
+		defer fd.Close()
+
+		var err error
+		f.C, err = ast.LoadCapabilitiesJSON(fd)
 		return err
 	}
-	defer fd.Close()
-	f.C, err = ast.LoadCapabilitiesJSON(fd)
-	return err
+
+	var errVersion error
+	f.C, errVersion = ast.LoadCapabilitiesVersionJSON(s)
+	if errVersion != nil && errPath != nil {
+		return fmt.Errorf("\n%v\nor\n%v", errPath, errVersion)
```

Comment:
Hmm is this error message comprehensible? 💭

Reply:
`opa capabilities --capabilities v0.99.0` would give the following output:

```
...
Flags:
      --capabilities string   set capabilities version or capabilities.json file path (default "")
      --current               print current capabilities in json
  -h, --help                  help for capabilities
      --versions              list capabilities versions

invalid argument "v0.99.0" for "--capabilities" flag: 
open v0.99.0: no such file or directory
or
no capabilities version found v0.99.0
```

Here, I have to make sure that both of errors are printed.

However, in case that the file is not a valid json, is covered on [L200](https://github.com/IoannisMatzaris/opa/blob/15ea6ba3ec9dc0379e5a6e5a88f4048dc379cfe4/cmd/flags.go#L200) and is returned on [L201](https://github.com/IoannisMatzaris/opa/blob/15ea6ba3ec9dc0379e5a6e5a88f4048dc379cfe4/cmd/flags.go#L201). Therefore, I could add a custom error on [L207](https://github.com/IoannisMatzaris/opa/blob/15ea6ba3ec9dc0379e5a6e5a88f4048dc379cfe4/cmd/flags.go#L207), like:

```suggestion
		return fmt.Errorf("no such file or capabilities version found %v", s)
```

What do you think of this approach?

Reply:
> What do you think of this approach?

This sounds good!

Reply:
[done](https://github.com/IoannisMatzaris/opa/commit/0dfe3e796d5e2d134f60533f68d9e6ba7794b746)